faq.html

```diff
@@ -164,141 +172,143 @@ <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="How_much_does_it_cos
-    <strong>seller</strong>:
-    <ul>
-      <li>0.003 BTC</li>
-      <li>0.600 LTC</li>
-      <li>0.150 DASH</li>
-    </ul>
+    <p>Bitcoin security deposits are necessary to protect Bisq traders from fraud and abuse.</p>
+    <p>Security deposits are set to be a percentage of the trade amount—2% by default—but they can be adjusted by the offer maker.</p>
 
-    <p>Bitcoin can be purchased via many channels: friends or family, Bitcoin meetups, Bitcoin ATMs, Vouchers, Localbitcoins or from any of the centralized exchanges.</p>
+    <p>We realize this requirement may be a barrier for bitcoin beginners, but there are plenty of ways to get your first bitcoin: friends and family, Bitcoin meetups, Bitcoin ATMs, vouchers, work for it, etc.</p>
   </div>
 
 
-  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="Does_Bisq_support_XYZ_payments">Does Bisq support ‘XYZ’ payments?</h6>
+  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="new-payment-accounts">Does Bisq support ‘XYZ’ payments?</h6>
   <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0">
 
-    <p>Bisq supports already a wide range of payment methods for national currencies. The criteria for inclusion are that the payment method is safe against chargebacks, and that it allows the arbitrator to view evidence of the transaction. National bank transfer
-      is supported in every country.</p>
+    <p>Bisq already supports a wide range of payment methods for national currencies, but new ones may be added as long as:</p>
+    <ul>
+      <li>chargebacks are unlikely</li>
+      <li>arbitrators can view evidence of the transaction</li>
+    </ul>
+    <p>National bank transfers are supported in every country.</p>
+    <p>You can suggest a new payment method on the <a href='https://bisq.community' target='_blank'>forum</a>, <a href='https://bisq.network/slack-invite' target='_blank'>Slack</a>, or <a href='https://github.com/bisq-network/bisq/issues' target='_blank'>GitHub</a>.</p>
   </div>
 
 
-  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="What_happens_in_case_of_software_bugs">What happens in case of software bugs?</h6>
+  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="software-bugs">What happens in case of software bugs?</h6>
   <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0">
 
     <p>If the Bisq application recognizes that you&#8217;re having a software problem, it will display a “support ticket” button. Your arbitrator will forward your bug report to the developers, who will begin an investigation of the problem. You may be asked
-      for the log file to assist the process.
+      for the log file to assist in the process.
     </p>
-    <p>If the button does not appear, you can open a support ticket by the key combination cmd + o (or ctrl + o).</p>
+    <p>If the button does not appear, you can open a support ticket by hitting Cmd+O (or Ctrl+O, depending on your system).</p>
```

Comment:
Maybe keep it lower case o, otherwise it looks like zero or users try uppercase O